frontend/components/Auction.vue

```diff
@@ -2,6 +2,13 @@
     <TextBlock :title="`Auction #${auctionId}`">
         <Alert v-if="errorText" class="my-3" :message="errorText" type="error" />
         <div v-if="auction">
+            <WalletBlock
+                v-if="!auction.isActive && !auction.isFinished && !walletAddress"
+                class="my-4"
+                :wallet-address="walletAddress"
+                :is-explanations-shown="isExplanationsShown"
+                @connectWallet="$emit('connect')"
+            />
```

Comment:
Can you please move this inside `RestartBlock`? Since it's already have logic to display or not display itself and the wallet connection is practically part of the restart button